apps/nextra/pages/en/build/move/compiler_v2.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Compiler v2 (beta)"
+---
+
+# Move Compiler v2 (beta)
+
+Move compiler v2 is a new tool which translates Move source code into Move bytecode. It unifies the architectures of the Move Compiler and the Move Prover, enabling faster innovation in the Move language. It also offers new for defining code optimizations which can be leveraged to generate more gas efficient code for Move programs.
+
+## Beta Testing
+
+Compiler v2 (version `2.0-unstable`) is currently in beta testing and the developer community is encouraged to try it. However, the code produced by this version of the compiler should **NOT** be deployed on `mainnet`. Using it for unit tests or for a local nodes, `devnet`, and `testnet` is fine. This page will be updated once this changes.
```

Comment:
Do we need a disclaimer for beta testing?

Reply:
I think what we have above is enough